components/studies/ui/src/main/resources/PhenoTips/StudyBindingClass.xml

```diff
@@ -58,7 +58,7 @@
 {{html clean="false" wiki="false"}}
 &lt;div class="form-template-option"&gt;
 &lt;label&gt;&lt;input type="radio" name="${prefix}${name}" value=""#if ("$!{value}" == '') checked="checked"#end/&gt; $services.localization.render('phenotips.studyBindingClass.noStudyOption')&lt;/label&gt;
-&lt;p class="xHint"&gt;$services.localization.render('phenotips.studyBindingClass.noStudyOption.hint')&lt;/p&gt;
+&lt;div class="xHint"&gt;$services.localization.render('phenotips.studyBindingClass.noStudyOption.hint')&lt;/div&gt;
```

Comment:
I think it looks a bit better with a `<p>` before the hint. Other than that, I couldn't find any regressions caused by this change, just somethings that were always there. Based on the comments on the ticket, I would otherwise approve.
![Minor Styleling](https://user-images.githubusercontent.com/46494177/56740854-b6dc4400-673f-11e9-8e5e-5f820dc52dd6.png)

Reply:
Good point, the spacing is a bit off now. I've improved it in the latest commit with some styling. I'd like to keep the `div` instead of `p` because:
* the study description container can't be `p`, as `p` elements can't contain other block elements that might be generated when rendering the description
* the container of the hint for "None" should be the same type of element as for study descriptions, for consistency.